scripts/svgGraphs.js

```diff
@@ -154,140 +154,142 @@ const {barLineGraph, stackedAreaGraph, pieGraph} = (function () {
         const data = Object.entries(config.data).filter(([,{x}]) => x && x.length > 0);
         const svg = d3.select("#" + id).append("svg")
             .attr("viewBox", [0, 0, width, height]);
-        const tooltip = d3.select("#" + id)
```

Comment:
this diff is silly, I just added an if around a huge chunk of code to avoid error when there is no data. Probably on a dev issue, but just to avoid errors on first or month or some edge case.

Reply:
<nod> Viewing the diff while ignoring whitespace is really useful for this MR!